apps/site/src/views/Landing/Landing.tsx

```diff
@@ -1,9 +1,37 @@
-import "./Landing.module.scss";
+"use client";
+import Container from "react-bootstrap/Container";
+import Row from "react-bootstrap/Row";
+
+import ApplyButton from "./components/ApplyButton/ApplyButton";
+import AboutSection from "./components/AboutSection/AboutSection";
+import MentorSection from "./components/MentorSection/MentorSection";
+import SponsorSection from "./components/SponsorSection/SponsorSection";
+import FAQSection from "./components/FAQSection/FAQSection";
+import styles from "./Landing.module.scss";
 
 export default function Landing() {
 	return (
 		<main>
-			<h1>ZotHacks 2023</h1>
+			<Container className="justify-content-center text-center">
+				<Row>
+					<h1>ZotHacks 2023</h1>
+				</Row>
+				<Row>
+					<ApplyButton />
+				</Row>
+				<Row>
+					<AboutSection />
+				</Row>
+				<Row>
+					<MentorSection />
+				</Row>
+				<Row>
+					<SponsorSection />
+				</Row>
+				<Row>
+					<FAQSection />
+				</Row>
```

Comment:
For these, it might be better to use the `Row` component within the component's code instead of here, especially since in most cases, Bootstrap grid components will be used in them.

Reply:
I feel like it would better to keep the here because that way the components are more containerized and scoped. If we need to have more grid components, we can define a new container within each component. At least, I think this way is better. If not, let me know and I'll change it!

Reply:
I added the rows to prevent the components from being inline. I thought it was a good way to separate each component in a singular column. If there is a better way or this is not our desired behavior, pls let me know!


### With Row
<img width="1045" alt="Screen Shot 2023-09-15 at 5 23 40 PM" src="https://github.com/HackAtUCI/zothacks-site-2023/assets/66880934/5017bf9f-97cd-43f2-95f7-8321d20cc3b2">

### Without Row
<img width="511" alt="Screen Shot 2023-09-15 at 5 26 05 PM" src="https://github.com/HackAtUCI/zothacks-site-2023/assets/66880934/ad8dde4b-086b-487a-a527-99d874dcc821">

Reply:
If the sections are actual elements and not just text nodes, they will stack vertically by default, no container or rows needed.